### PR TITLE
Daily analytics report (based on monthly report)

### PR DESCRIPTION
**Pull request by pascalheidmann-bedarf**
### 1. Why is this change necessary?
More insight

### 2. What does this change do, exactly?
Adds new report for daily data instead of compressed to a single month

### 6. Checklist

- [x] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: engine/Shopware/Models/Analytics/Repository.php [hunk 1]
```diff
@@ -783,6 +783,52 @@ public function getAmountPerMonth(\DateTimeInterface $from = null, \DateTimeInte
...
+     */
+    public function getAmountPerDay(\DateTimeInterface $from = null, \DateTimeInterface $to = null, array $shopIds = [])
+    {
+        $dateCondition = 'DATE_FORMAT(ordertime, \'%Y-%m-%d\')';
+        $builder = $this->createAmountBuilder($from, $to, $shopIds)
+            ->addSelect($dateCondition . ' AS date')
+            ->groupBy($dateCondition)
+            ->orderBy('date', 'DESC');
+
+        $builder = $this->eventManager->filter('Shopware_Analytics_AmountPerDay', $builder, [
```
Comment (shyim) [inline engine/Shopware/Models/Analytics/Repository.php:825]: Maybe extract the builder in another method xxxBuilder and call the method here like in all other repositories, instead creating a new event

Comment (pascalheidmann-bedarf) [reply engine/Shopware/Models/Analytics/Repository.php:825]: In every other repository I would have done exactly that.
But as this repo is already a collection of functions that are all created like the new one - builder + event - changing this pattern wouldn't be beneficial.

**Comment by shyim** [comment]
Hey,
the table layout does not make sense to me. Did you forget something? (Snippet in Diagram for day is also missing)
2019-10-07_15-45-03-1174x575

**Comment by pascalheidmann-bedarf** [comment]
@shyim rebased to current 5.6 branch and hopefully now replaced all old date formatting rules...

**Comment by shyim** [comment]
It looks still the same in table mode

**Comment by shyim** [comment]
Can i look again?

**Comment by pascalheidmann-bedarf** [comment]
@shyim I found another instance but have broken my local testsystem right now so I haven't checked if every instance got fixed now 🙈 But should be the last one, at least for the table view.

**Comment by shyim** [comment]
Hey,
what you built exists already :see_no_evil: . Its called "Fast overview", it's the first menu point
image

So i will close this PR.

**Pull request closed without merging**